playbooks/vars/maas-verify.yml

```diff
@@ -18,3 +18,7 @@
 #
 maas_verify_pip_packages:
   - futures
+  - pyyaml
+  - rackspace-monitoring-cli
+  - requests
+  - waxeye
```

Comment:
Can `waxeye` be removed from playbooks/vars/maas-agent.yml ?  I'd imagine that is only used by rpc-maas-tool.py in the verify.

Reply:
Done